compute/admin_guide/waas/waas_api_observation.adoc

```diff
@@ -61,7 +61,7 @@ NOTE: If a rule with an app is deleted from the WAAS policy, its learned endpoin
 The *Path risk factors* indicate the three critical path risks identified in the endpoints:
 * Endpoints accessible from the internet (icon).
 * Endpoints that do not require authentication (icon) to be accessed.
-* Endpoints with sensitive data (icon) such as Credit card, PII, ID query parameter, and Session cookies, .etc. The sensitive data is defined under *Defend > WAAS > Sensitive data* as described in the https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/waas/log_scrubbing[WAAS sensitive] page.
+* Endpoints with sensitive data (icon) such as Credit card, PII, ID query parameter, and Session cookies, .etc. The xref:log-scrubbing.adoc[sensitive data] is defined under *Defend > WAAS > Sensitive data*.
```

Comment:
"log-scrubbing.adoc" > The filename has an underscore, like so: "log_scrubbing.adoc".